workflows/ampliseq.nf

```diff
@@ -420,6 +423,26 @@ workflow AMPLISEQ {
         ch_dada2_asv = FILTER_CODONS.out.asv
     }
 
+    //
+    // MODULE : ASV post-clustering with VSEARCH
+    //
+    if (params.vsearch_cluster) {
+        ch_fasta_for_clustering = ch_dada2_fasta
+            .map {
+                fasta ->
+                    def meta = [:]
+                    meta.id = "ASV_post_clustering"
+                    [ meta, fasta ] }
+        VSEARCH_CLUSTER ( ch_fasta_for_clustering )
+        ch_versions = ch_versions.mix(VSEARCH_CLUSTER.out.versions.ifEmpty(null))
+        FILTER_CLUSTERS ( VSEARCH_CLUSTER.out.clusters, ch_dada2_asv )
+        ch_versions = ch_versions.mix(FILTER_CLUSTERS.out.versions.ifEmpty(null))
+        MERGE_STATS_CLUSTER ( ch_stats, FILTER_CLUSTERS.out.stats )
+        ch_stats = MERGE_STATS_CLUSTER.out.tsv
+        ch_dada2_fasta = FILTER_CLUSTERS.out.fasta
+        ch_dada2_asv = FILTER_CLUSTERS.out.asv
+    }
+
```

Comment:
Are you sure thats the right position? After all filtering, such as kingdom, length, codons took place? Or rather before? Not sure. But in any case, the sequence of filtering should be represented and mentioned in the documentation. For other filters the sequence of filtering steps might not matter, but here it matters I think.

Reply:
That's a very good point. Now that I've thought about it more, it probably makes more sense to place this before the other filter steps.